src/react-app-env.d.ts

```diff
@@ -1,2 +1,9 @@
-// eslint-disable-next-line
-/// <reference types="react-scripts" />
+interface People {
```

Comment:
The name People doesn't correspond to sense. It's maybe better to name Person.

Reply:
Yep, you have only 1 person here.